src/khoj/routers/helpers.py

```diff
@@ -1477,10 +1478,16 @@ def get_user_config(user: KhojUser, request: Request, is_detailed: bool = False)
 
     user_subscription_state = get_user_subscription_state(user.email)
     user_subscription = adapters.get_user_subscription(user.email)
+
     subscription_renewal_date = (
         user_subscription.renewal_date.strftime("%d %b %Y")
         if user_subscription and user_subscription.renewal_date
-        else (user_subscription.created_at + timedelta(days=7)).strftime("%d %b %Y")
+        else None
```

Comment:
Should this continue to be set to `user_subscription.created_at + timedelta(days=7)).strftime("%d %b %Y")` if `renewal_date` is unset? The downstream dependencies e.g common/auth.ts, settings/page.tsx on web app) seem to expect `subscription_renewal_date` to be a (non-nullable) string

Reply:
I don't think so. It's a bit misleading, because it suggests the user is enrolled in something when they're not. This should correctly indicate that the user has no active enrollment at the time.